workflows/funcscan.nf

```diff
@@ -174,6 +176,41 @@ workflow FUNCSCAN {
         ch_taxonomy_tsv = Channel.empty()
     }
 
+    /*
+        FUNCTION
+    */
+    if (params.run_function_interproscan) {
+        def filtered_faas = ch_prepped_input.faas.filter { meta, file ->
+            if (file != [] && file.isEmpty()) {
+                log.warn("[nf-core/funcscan] Annotation of the following sample produced an empty FAA file. InterProScan classification of the CDS requiring this file will not be executed: ${meta.id}")
+            }
+            !file.isEmpty()
+        }
+
+        SEQKIT_SEQ_FILTER(filtered_faas)
+        ch_versions = ch_versions.mix(SEQKIT_SEQ_FILTER.out.versions)
+        ch_input_for_function =  SEQKIT_SEQ_FILTER.out.fastx
+
+        FUNCTION (
+            ch_input_for_function
+        )
+        ch_versions = ch_versions.mix(FUNCTION.out.versions)
+
+        ch_interproscan_tsv = FUNCTION.out.tsv.map { meta, file ->
+            if (file == [] || file.isEmpty()) {
+                log.warn("[nf-core/funcscan] Functional annotation with INTERPROSCAN produced an empty TSV file. Passing empty file for ${meta.id}.")
```

Comment:
What does 'Passin'g mean here? Skipping?

Reply:
is it better now?